server/routes/request.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+
+const {
+    getRequests,
+    addRequest,
+    updateRequest,
+} = require('../controllers/request');
+
+router.route('/getRequests').get(protect, getRequests);
```

Comment:
Please use hyphenated case in route names with multiple words.

Eg: `post-requests`